tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-torch181-cuda102.yml

```diff
@@ -3,7 +3,7 @@ trigger: none
 stages:
 - template: templates/py-packaging-training-cuda-stage.yml
   parameters:
-    build_py_parameters: --enable_training --update --build
+    build_py_parameters: --enable_training --update --build --enable_training_torch_interop --cmake_extra_defines onnxruntime_BUILD_UNIT_TESTS=OFF
```

Comment:
So this "--cmake_extra_defines onnxruntime_BUILD_UNIT_TESTS=OFF" doesn't work?

Reply:
No. It doesn't. A potential fix (trying):
```
    parser.add_argument(
        "--cmake_extra_defines", action='append', nargs="+",
        help="Extra definitions to pass to CMake during build system "
        "generation. These are just CMake -D options without the leading -D.")
```
We missed `action=append`.